expression/builtin_string_vec_test.go

```diff
@@ -56,6 +56,11 @@ var vecBuiltinStringCases = map[string][]vecExprBenchCase{
 	ast.ConcatWS: {},
 	ast.Convert:  {},
 	ast.Substring: {
+		{
```

Comment:
Please post the benchmark result in this PR's description.

Reply:
I run the benchmark, but get the follow message:

    $go test -v -benchmem -bench=BenchmarkVectorizedBuiltinStringFunc -run=BenchmarkVectorizedBuiltinStringFunc -args "builtinSubstringBinary2ArgsSig"
	goos: darwin
	goarch: amd64
	pkg: github.com/pingcap/tidb/expression
	BenchmarkVectorizedBuiltinStringFunc-4   	1000000000	         0.0266 ns/op	       0 B/op	       0 allocs/op
	PASS
	ok  	github.com/pingcap/tidb/expression	0.288s